scripts/mkcloud

```diff
@@ -1381,6 +1386,14 @@ function sanity_checks()
             complain 80 "Please set nodenumberlonelynode."
         fi
     fi
+
+    case $cinder_backend in
+        ''|netapp|local|raw|rbd)
```

Comment:
I don't see how `local`, `raw` and `rbd` can be configured now. Or are the defaults fine?